nisystemlink/clients/auth/models/_auth_models.py

```diff
@@ -0,0 +1,373 @@
+# generated by datamodel-codegen:
+#   filename:  <stdin>
+
+from __future__ import annotations
+
+from datetime import datetime
+from enum import Enum
+from typing import Any, Dict, List, Optional
+
+from nisystemlink.clients.core._uplink._json_model import JsonModel
+from pydantic import Field
+
+
+class AuthStatement(JsonModel):
+    actions: Optional[List[str]] = None
+    """
+    A list of actions the user is allowed to perform
+    """
+    resource: Optional[List[str]] = None
+    """
+    A list of resources the user is allowed to access
+    """
+    workspace: Optional[str] = Field(None, example="workspace-id")
```

Comment:
Do you know if the `example` parameter actually does anything in Pydantic?

Reply:
Useful only when exporting the Pydantic model as JSON Schema and hence not useful in the scope of the nisystemlink-clients module.